app/javascript/App/bundles/ExchangeListings/components/ListingsList.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import ListingItem from './ListingItem'
+
+class ListingsList extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      listings: props.initialListings,
+      detectedSince: null,
+    }
+  }
+
+  onComponentMount() {
+    this.fetchListings()
```

Comment:
this method appears undefined. perhaps this is stubbed here for a method coming later?